notebooks/406-human-pose-estimation-3d/406-human-pose-estimation-3d.ipynb

```diff
@@ -0,0 +1,573 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9de9a93e-9247-4799-a5bb-2ec1575ae8c2",
+   "metadata": {},
+   "source": [
+    "# Live 3D Human Pose Estimation with OpenVINO\n",
+    "\n",
+    "This notebook demonstrates a live 3D Human Pose Estimation webcam example with OpenVINO. We utilize the model [human-pose-estimation-3d-0001](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/human-pose-estimation-3d-0001) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/). At the last session of this notebook, you will see the live inference results from your webcam (if available). Alternatively, you can also upload a video file to test out the algorithms.\n",
```

Comment:
Suggested changes : " This notebook demonstrates a 3D display for human pose estimation with live webcam inferencing with OpenVINO. "